src/main/java/techreborn/items/ItemUpgrade.java

```diff
@@ -64,15 +54,4 @@ public void process(
 			ItemStack stack) {
 		behavior.process(blockEntity, handler, stack);
 	}
-
-	@Environment(EnvType.CLIENT)
-	@Override
-	public void appendTooltip(ItemStack stack, @Nullable World world, List<Text> tooltip, TooltipContext context) {
-		if(stack.getItem() == TRContent.Upgrades.SUPERCONDUCTOR.item){
-			if(Screen.hasShiftDown()){
-				tooltip.add(new LiteralText(Formatting.GOLD + "Blame obstinate_3 for this"));
```

Comment:
Can we keep this please. I know its an "in joke" but its something that I would like to keep in the mod. Thanks.